about/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Doodle Dandy | Doodle Dandy</title>
+    <link rel="stylesheet" href="../assets/vendor/simplecss/simple.min.css" />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/fontawesome.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/brands.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/vendor/fontawesome/6.2.1/css/solid.min.css"
+    />
+    <link rel="stylesheet" href="../assets/styles/main.css" />
+    <link
+      rel="shortcut icon"
+      href="../assets/images/favicon.ico"
+      type="image/x-icon"
+    />
+  </head>
+
+  <body>
+    <header>
+      <img src="../assets/images/header.png" alt="Doodle Dandy" />
+      <nav>
+        <a href="/">Home</a>
+        <a href="/bernedoodle">Bernedoodle Info</a>
+        <a href="/breed">AussieDoodle Info</a>
+        <a href="/about">About us</a>
+        <a href="/current" class="current">Current Litter</a>
```

Comment:
For the `nav` section on each page, you need to move the `class="current"` attribute to the link for the active page. So for this one, it should move to the `/about` link.

```suggestion
        <a href="/about" class="current">About us</a>
        <a href="/current">Current Litter</a>
```

Reply:
THATS how to fix the damn yellow box haha